Pachanga/HomeView.swift

```diff
@@ -35,7 +35,12 @@ struct HomeView: View {
                                     .font(.title.bold())
                                 
                                 // Session
-                                SessionCard(playersTitle: "Javi, Danielle, Jorge & 2 más", startTime: "8:00", endTime: "9:30", location: "Club Vóley Playa Muchavista")
+                                NavigationLink {
+                                    SessionView()
```

Comment:
Can't we already pass some properties into SessionView for each card instead of the hardcoded data?